Dev/GitHub/github-review-requests-TS-5m.ts

```diff
@@ -12,13 +12,16 @@
 //  <xbar.var>string(USERNAME=""): Github User name.</xbar.var>
 //  <xbar.var>string(ORGANIZATION=""): Filter PRs by Organization name. (optional)</xbar.var>
 //  <xbar.var>string(WIP_FILTER=""): Title included in a Work in Progress PR to exclude from the list. (optional)</xbar.var>
+//  <xbar.var>string(API_ENDPOINT=""): URL to your GitHub GraphQL Endpoint. (optional)</xbar.var>
```

Comment:
Rather than list it as optional, consider having the main GitHub URL as the default value.

```suggestion
//  <xbar.var>string(API_ENDPOINT="https://api.github.com/graphql"): URL to your GitHub GraphQL Endpoint.</xbar.var>
```